frontend/src/utils/SpeakingDetector/speakingDetector.ts

```diff
@@ -77,7 +77,11 @@ class SpeakingDetector extends EventEmitter {
    */
   turnSpeakingDetectorOn = async () => {
     try {
+      if (!this.selectedMicrophoneId) {
+        return;
+      }
```

Comment:
how come?

Reply:
I started after the changes I've done that the speaking detector was returning undefined. the useSpeakingIndicator hook gets called a few times so it gets initialized okay but adding this `if` statement removes a console log error